### PR TITLE
[12.0][FIX] Form view of mandate had 2 "type" fields

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

File: account_banking_sepa_direct_debit/views/account_banking_mandate_view.xml [hunk 1]
```diff
@@ -17,12 +17,13 @@
...
             <field name="scheme" attrs="{'invisible': [('format', '!=', 'sepa')],
                                          'required': [('format', '=', 'sepa')]}"/>
-            <field name="type" attrs="{'invisible': [('format', '!=', 'sepa')],
-                                       'required': [('format', '=', 'sepa')]}"/>
             <field name="recurrent_sequence_type"
                    attrs="{'invisible': ['|', ('type', '=', 'oneoff'), ('format', '!=', 'sepa')],
                            'required': [('type', '=', 'recurrent')]}"/>
         </field>
+        <field name="type" position="attributes">
+            <attribute name="attrs">{'required': [('format', '=', 'sepa')]}</attribute>
```
Comment (pedrobaeza) [inline account_banking_sepa_direct_debit/views/account_banking_mandate_view.xml:25]: What about the `'invisible': [('format', '!=', 'sepa')]` part?

Comment (alexis-via) [reply account_banking_sepa_direct_debit/views/account_banking_mandate_view.xml:25]: Well, type field is now part of account_banking_mandate. So I would be strange to hide it in an upper-level module.

Comment (pedrobaeza) [reply account_banking_sepa_direct_debit/views/account_banking_mandate_view.xml:25]: Yes, makes sense

**Review by pedrobaeza — APPROVED** [review]

**Review by HaraldPanten — APPROVED** [review]

**Pull request merged**